Makefile

```diff
@@ -236,9 +234,7 @@ check-pytest: all tests/buttons.png tests/ocr/menu.png
 	              -e tests/auto-selftest-example-test-pack/tests/empty_dir/subdir/example_with_no_tests.py \
 	              -e tests/vstb-example-html5/ \
 	              -e tests/webminspector/ \
-	              -e vendor/) \
-	    nosetest-issue-49-workaround-stbt-control.py && \
-	rm nosetest-issue-49-workaround-stbt-control.py
```

Comment:
So now we aren't running `stbt-control::test_main`. I'm not sure what the point of that test is. Might as well remove it.

Reply:
Done